run_dir/design/flowcells.html

```diff
@@ -184,8 +184,17 @@ <h1><span id="illumina_page_title">Illumina flowcells</span>
                                         {% if 'lane_info' in flowcells[onefc] %}
                                             {% for onelane in flowcells[onefc]['lane_info'] %}
                                                 {% if flowcells[onefc]['lane_info'][onelane].get('er_rate') %}
+                                                    {% if 'MiSeq' in flowcells[onefc].get('run_mode', '') %}
+                                                        {% if run_setup > 150 %}
+                                                            {% set er_rate_threshold = 5 %}
+                                                        {% else %}
+                                                            {% set er_rate_threshold = 2 %}
+                                                        {% end %}
+                                                    {% else %}
+                                                        {% set er_rate_threshold = 2 %}
+                                                    {% end %}
```

Comment:
```suggestion
                                                    {% if 'MiSeq' in flowcells[onefc].get('run_mode', '') and run_setup > 150 %}
                                                        {% set er_rate_threshold = 5 %}
                                                    {% else %}
                                                        {% set er_rate_threshold = 2 %}
                                                    {% end %}
```

Would this work?

Reply:
Hi, no. I tested it previously and it didn't work.